### PR TITLE
correct the CHANGELOG-1.11.md

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind cleanup
/sig release

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by qingsenLi** [comment]
/assign @liggitt 
PTAL

File: CHANGELOG-1.11.md [hunk 1]
```diff
@@ -1179,7 +1178,7 @@ controllerManagerExtraVolumes:
...
 * The old dynamic client has been replaced by a new one.  The previous dynamic client will exist for one release in `client-go/deprecated-dynamic`.  Switch as soon as possible. ([#63446](https://github.com/kubernetes/kubernetes/pull/63446), [@deads2k](https://github.com/deads2k))
 * In-tree support for openstack credentials is now deprecated. please use the "client-keystone-auth" from the cloud-provider-openstack repository. details on how to use this new capability is documented [here](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-client-keystone-auth.md) ([#64346](https://github.com/kubernetes/kubernetes/pull/64346), [@dims](https://github.com/dims))
-* The GitRepo volume type is deprecated. To provision a container with a git repo, mount an `EmptyDir` into an `InitContainer` that clones the repo using git, then moEmptyDir` into the Pod's container.
+* The GitRepo volume type is deprecated. To provision a container with a git repo, mount an `EmptyDir` into an `InitContainer` that clones the repo using git, then `moEmptyDir` into the Pod's container.
```
Comment (liggitt) [inline CHANGELOG-1.11.md]: this is a typo, should be "...then mount the `EmptyDir`"

**Comment by qingsenLi** [comment]
/remove-sig api-machinery apps cli cloud-provider cluster-lifecycle network node testing
/remove-kind api-change

**Comment by qingsenLi** [comment]
/remove-area apiserver cloudprovider dependency kubectl kubelet release-eng test

**Comment by qingsenLi** [comment]
/cc @liggitt 
fixed the PR,thanks a lot

**Comment by qingsenLi** [comment]
/assign @liggitt
PTAL，thanks

**Pull request merged**